tests/test_compositional_attention.py

```diff
@@ -16,19 +16,18 @@
 )
 
 BATCH = 2
-SEQ = 128 if torch.cuda.is_available() else 32
-MODEL = 128 if torch.cuda.is_available() else 64
+SEQ = 128 if torch.cuda.is_available() else 16
+MODEL = 128 if torch.cuda.is_available() else 32
 
 assert ATTENTION_REGISTRY.keys(), "Attention layers should have been registered"
 
 
+@pytest.mark.parametrize("heads", [1, 4])
 @pytest.mark.parametrize("attn_dropout", [0.0, 0.3])
 @pytest.mark.parametrize("causal", [True, False])
-@pytest.mark.parametrize("heads", [1, 4])
 @pytest.mark.parametrize("rules", [1, 4])
 @pytest.mark.parametrize("q_compose", [False, True])
 @pytest.mark.parametrize("dim_selection", [MODEL // 2, None])
-@pytest.mark.parametrize("bias", [True, False])
```

Comment:
the bias part is not well tested per say, and it's just something we pass to the constructor (so it's tested / config typing already). Removing this halves the number of tests of course..